### ISSUE TITLE
Add tooltip for disabled images

### ISSUE THREAD

**Issue by murisceman** [pruvious/pruvious#44]
**Location:** Media library

When an image does not meet the requirements (width, height, format, etc.) for selection in an `image` field, it is currently grayed out. To improve the editor UX, we could add a tooltip that displays additional details when hovering over the disabled image.

**Comment by andreasvirkus** [comment]
Hi @murisceman, if it's alright I'd like to tackle this one. I can't get the playground up and running locally though, it results in a 500 error that doesn't get logged I think

```sh
pnpm dev

> pruvious-playground@ dev /Users/andreasvirkus/code/personal/pruvious/playground
> nuxi dev

Nuxt 3.12.4 with Nitro 2.9.7                                                                                                                                                                        6:57:07 AM
[get-port] Unable to find an available port (tried 3000 on host "localhost"). Using alternative port 3001.                                                                                          6:57:07 AM
                                                                                                                                                                                                    6:57:07 AM
  ➜ Local:    http://localhost:3001/
  ➜ Network:  use --host to expose

  ➜ DevTools: press Shift + Option + D in the browser (v1.3.7)                                                                                                                                      6:57:09 AM

✔ Database synced in 91 ms                                                                                                                                                                pruvious 6:57:10 AM
✔ Pruvious built in 885 ms                                                                                                                                                                pruvious 6:57:10 AM

 ERROR  Pre-transform error: failed to resolve "extends":"./.nuxt/tsconfig.json" in /Users/andreasvirkus/code/personal/pruvious/tsconfig.json                                                       6:57:11 AM

ℹ Vite server warmed up in 1031ms                                                                                                                                                                  6:57:12 AM
ℹ Vite client warmed up in 1754ms                                                                                                                                                                  6:57:13 AM
✔ Nuxt Nitro server built in 2066 ms                                                                                                                                                         nitro 6:57:13 AM
```

![Screenshot 2024-08-08 at 06 57 37](https://github.com/user-attachments/assets/6c420a75-f088-42c9-8463-c6435fb1173b)

**Comment by murisceman** [comment]
Hey @andreasvirkus, thank you for offering your help. I appreciate it! The thing is, I've already done this in 2ba9e8c :D

To address the issue you were facing, you need to run `pnpm dev:prepare` to make it work. I've just included it into `postinstall`, so it will execute automatically after running `pnpm i`.

**Comment by murisceman** [comment]
Also, I'll publish this change in a new release once more features are completed, as it's not a critical bug.

**Comment by andreasvirkus** [comment]
Hehe, I just arrived at the same discovery 😅 

![Screenshot 2024-08-08 at 20 32 47](https://github.com/user-attachments/assets/e6cfc6fd-c505-460d-9226-70da31637828)


Awesome, feel free to close this out then and thanks for working so fast!

**Comment by murisceman** [comment]
Released in `v3.12.5`.
